GTMAppAuthSwift.podspec

```diff
@@ -0,0 +1,47 @@
+Pod::Spec.new do |s|
+
+  s.name          = "GTMAppAuthSwift"
+  s.version       = "1.3.0"
+  s.swift_version = "4.0"
+  s.summary       = "Authorize GTM Session Fetcher requests with AppAuth via GTMAppAuth"
+
+  s.description   = <<-DESC
+
+GTMAppAuth enables you to use AppAuth with the Google Toolbox for Mac - Session
+Fetcher and Google APIs Client Library for Objective-C For REST libraries by
+providing an implementation of GTMFetcherAuthorizationProtocol for authorizing
+requests with AppAuth.
+
+                    DESC
+
+  s.homepage      = "https://github.com/google/GTMAppAuth"
+  s.license       = { :type => "Apache", :file => "LICENSE" }
+  s.author        = "Google LLC"
+
+  s.source        = { :git => "https://github.com/google/GTMAppAuth.git", :tag => s.version }
+  s.prefix_header_file = false
+  s.source_files = "GTMAppAuthSwift/Sources/*.swift"
+
+  ios_deployment_target = "9.0"
```

Comment:
Have we verified that these min versions are correct for the Swift library?

Reply:
I believe so. I think a great validation step to do at the end is to deploy our example app to the earliest versions possible.

https://developer.apple.com/documentation/swift#2984801